craft_parts/plugins/cmake_plugin.py

```diff
@@ -102,11 +111,6 @@ def get_build_commands(self) -> List[str]:
             f'"{self._part_info.part_src_subdir}"',
             "-G",
             f'"{options.cmake_generator}"',
-            # Install on a location we search when building using staged files
-            # (e.g. CMAKE_PREFIX_PATH/lib, CMAKE_PREFIX_PATH/lib/<arch> for libs,
-            # see https://cmake.org/cmake/help/latest/command/find_library.html).
-            # This can be overridden by the user using cmake-parameters.
-            "-DCMAKE_INSTALL_PREFIX=",
         ] + options.cmake_parameters
```

Comment:
we could detect that -DCMAKE_INSTALL_PREFIX is not defined an emit a warning here

Reply:
This parameter is useful when building libraries that will be staged and used by another part that also uses the cmake plugin. This is a very specific case and the majority of use cases of the cmake plugin won't need it, but the warning would be emited anyway leading to some confusion as to when the parameter is really needed. Using external documentation in this case would allow us to explain the scenario in more detail.